packages/cli/src/util/parse-env.ts

```diff
@@ -0,0 +1,34 @@
+// Converts `env` Arrays, Strings and Objects into env Objects.
+import { Dictionary } from '@vercel/client';
+
+export const parseEnv = (env?: string[] | Dictionary<string>) => {
+  if (!env) {
+    return {};
+  }
+
+  if (typeof env === 'string') {
+    // a single `--env` arg comes in as a String
+    env = [env];
+  }
```

Comment:
surprised typescript isn't catching this... make sure to update the `env` function parameter type to include `string`